docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4705,8 +4710,18 @@ controller populates the `MasterArbitrationUpdate` field in this message using
 its `role_id` and `election_id` and the `device_id` of the device, as explained
 in detail in the [Master-Slave Arbitration and Controller
 Replication](#sec-master-slave-arbitration-and-controller-replication)
-section. For any given (`device_id`, `role_id`), the controller with the highest
-`election_id` is the master and the rest are slaves.
+section. For any given (device_id, role_id), the P4Runtime server keeps track
+of the highest election_id that it has ever received. If a controller's
+election_id is equal to the highest election_id that the server has ever
```

Comment:
Is there an assumption that no two clients will ever have the same election_id for the same (`device_id`, `role_id`) pair?

If so, is this a good place to state that explicitly?

Reply:
Done.